frontend/src/Editor/Components/ButtonGroup.jsx

```diff
@@ -34,20 +34,21 @@ export const ButtonGroup = function Button({
   };
 
   const [defaultActive, setDefaultActive] = useState(defaultSelected);
-  const [data, setData] = useState(
-    // eslint-disable-next-line no-unsafe-optional-chaining
-    values?.length <= labels?.length ? [...labels, ...values?.slice(labels?.length)] : labels
-  );
-  // data is used as state to show what to display , club of label+values / values
+  const [data, setData] = useState(values);
+
   useEffect(() => {
     setDefaultActive(defaultSelected);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [JSON.stringify(defaultSelected)]);
 
   useEffect(() => {
+    setData(values);
```

Comment:
```suggestion
```

Reply:
> 
Below this line we are checking for length of labels vs values and setting data to labels if length is more